frontend/src/metabase/dashboard/components/DashCard.jsx

```diff
@@ -144,6 +159,10 @@ export default class DashCard extends Component {
       !isEditing &&
       mainCard.visualization_settings["dashcard.background"] === false;
 
+    const isEditingDashboardLayout =
+      isEditing &&
+      !(clickBehaviorSidebarDashcard != null || isEditingParameter);
```

Comment:
I think the double negation here (the `!(foo != null)` part) is a little confusing. Can you change it to this:

```
    const isEditingDashboardLayout = isEditing &&
        clickBehaviorSidebarDashcard == null &&
        !isEditingParameter;
```

That way it sorta reads, "`isEditingDashboardLayout` is `true` when we're editing, there's no `clickBehaviorSidebarDashcard`, and I'm not editing a parameter"

Reply:
I agree that it reads easier this way, thanks!